easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -161,6 +161,13 @@ def configure_step(self):
                  r"\1\n\techo 'libfox: external module used' #"),
             ]
             apply_regex_substitutions('Makefile', regex_subs)
+            # Make configure and make look in the correct directory
+            regex_subs = [
+                (r"\(TOPDIR\)/FoX/finclude",
+                 r"(EBROOTFOX)/finclude"),
+            ]
+            apply_regex_substitutions('install/configure', regex_subs)
+            apply_regex_substitutions('make.inc', regex_subs)
```

Comment:
Do you also need to patch `make.inc`? Doesn't seem to make any difference w.r.t. `FoX` include dir?

Reply:
The reference is to `-L$(TOPDIR)/FoX/lib`, and it's not getting patched currently:

```
$ grep TOPDIR make.inc* | grep FoX
make.inc:FOX_LIB  = -L$(TOPDIR)/FoX/lib  -lFoX_dom -lFoX_sax -lFoX_wxml -lFoX_common\
make.inc.orig.eb:FOX_LIB  = -L$(TOPDIR)/FoX/lib  -lFoX_dom -lFoX_sax -lFoX_wxml -lFoX_common\
```

Maybe simplify the regex to make it just replace `\(TOPDIR\)/FoX/` with `(EBROOTFOX)/` ?

Reply:
@damianam Ping on fixing this?